Framework/Sources/FloatingPanel.swift

```diff
@@ -49,6 +49,11 @@ class FloatingPanel: NSObject, UIGestureRecognizerDelegate {
     var interactionInProgress: Bool = false
     var isDecelerating: Bool = false
 
+    // Animation handling
+    private var alreadyMovedTranslationOfPan: CGPoint = .zero
+    private var totalYTranslation: CGFloat = 0
+    private var nextState: FloatingPanelPosition = .tip
```

Comment:
We should think about wrapping these three properties a `struct` that is only available while a pan gesture is active. We could go a step further and add the pan gesture itself to the `struct` to compute some of these properties when they are accessed.

What do you think?

Reply:
Wrapping those properties in a `struct` is nicer but I'm not sure if it adds that much value. 
The `UIPanGestureRecognizer` in `FloatingPanel.swift` is of type `FloatingPanelPaneGesutreRecognizer` which has a delegate which a has a required type of `FloatingPanel`. Therefore, I'm not sure if adding the pan gesture to the struct is not a little bit unintuitive.